common/app/controllers/EmailSignupController.scala

```diff
@@ -78,13 +65,17 @@ class EmailSignupController(wsClient: WSClient, val controllerComponents: Contro
   }
 
   def renderForm(emailType: String, listId: Int): Action[AnyContent] = Action { implicit request =>
-    Cached(1.day)(RevalidatableResult.Ok(views.html.emailFragment(emailLandingPage, emailType, listId)))
+    val identityName = EmailNewsletter(listId).map(_.identityName)
```

Comment:
Will this continue to work after V2 newsletter IDs are added to `EmailNewsletters`?  Do we need to support aliases (or equivalent)?